quickstart-templates/workload-identity-creates-workload-identities/main.bicep

```
@@ -0,0 +1,86 @@
+targetScope = 'tenant'
+extension microsoftGraph
+
+
+@description('The display name for the application.')
+param applicationIdentityRegistrationDisplayName string
+
+@description('The name for the application.')
+param applicationIdentityRegistrationName string
+
+@description('The owner of the organization that it assigned to a workload identity that is used by GitHub Actions to deploy further resources.')
+param gitHubOwner string
+
+@description('The GitHub repository that is assigned to a workload identity that is used by GitHub Actions to deploy further resources.')
+param gitHubRepo string
+
+@description('Subject of the GitHub Actions workflow\'s federated identity credentials (FIC) that is checked before issuing an Entra ID access token to access Azure resources. GitHub Actions subject examples can be found in https://docs.github.com/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#example-subject-claims.')
+var gitHubActionsFederatedIdentitySubject = 'repo:${gitHubOwner}/${gitHubRepo}:ref:refs/heads/main'
+
+
+// The githubOIDCProvider and microsoftEntraAudience are well-known values that are used to
+// create the federated identity credentials for the GitHub Actions application so the two systems
+// can properly authenticate and authorize between each other. For other OIDC enabled systems you need to check
+// their documentation to find out the correct values so the federated identity credentials can be created
+// to connect to Azure.
+var githubOIDCProvider = 'https://token.actions.githubusercontent.com'
+var microsoftEntraAudience = 'api://AzureADTokenExchange'
+
+// In order for the Azure and GitHub Actions to communicate with each other, an
+// application for GitHub Actions is created to Azure Entra. This application is then
```

```suggestion
// application for GitHub Actions is created in Microsoft Entra. This application is then
```
Entra is not part of Azure (any more) - yes it used to be Azure AD, but part of the rebranding was to decouple from Azure, as we are pushing Entra to be a multi-cloud offering (and not tied to Azure in any way).